ci/check_style.sh

```diff
@@ -16,5 +16,13 @@ rapids-dependency-file-generator \
 rapids-mamba-retry env create --yes -f env.yaml -n checks
 conda activate checks
 
+RAPIDS_VERSION="$(cat "${BASH_SOURCE%/*}/../RAPIDS_VERSION")"
```

Comment:
A possible alternative to get this path is to use something like `"$(git rev-parse --show-toplevel)/RAPIDS_VERSION"`. We use that approach in a few places, including `cmake/scripts/run-cmake-format.sh` in this PR.